src/js/routes/services.js

```diff
@@ -13,7 +13,7 @@ let serviceRoutes = {
   children: [
     {
       type: Route,
-      name: 'services-deployments',
+      name: '-services-deployments',
```

Comment:
Thanks for the nice explanation. Yeah, I can't think of anything better in-place, so I'm OK with it.